taskgraph.json

```diff
@@ -30,12 +30,12 @@
           "image": "quay.io/mozilla/gaia-taskenv:0.8.9",
           "command": [
             "entrypoint",
-            "echo 'what' && node tests/taskcluster/bin/graph > /graph.json && node tests/taskcluster/bin/raptor_graph > /raptor_graph.json && cat /graph.json && cat /raptor_graph.json"
+            "echo 'what' && node tests/taskcluster/bin/graph > /graph.json && node tests/taskcluster/bin/raptor_decision > /raptor_decision.json && cat /graph.json && cat /raptor_decision.json"
```

Comment:
Ah yes good point, updated the raptor_decision.yml and added it to tasks.yml for bin/graph.